docs/docs/developers/contracts/writing_contracts/storage/notes.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Using Notes
+---
+
+Some optionally background resources on notes can be seen here:
+- [High level network architecture](../../../../learn/about_aztec/technical_overview#high-level-network-architecture), specifically the Private Execution Environment
+- [Transaction lifecycle (simple diagram)](../../../../learn/concepts/transactions#simple-example-of-the-private-transaction-lifecycle)
+- [Public and Private state](../../../../learn/concepts/hybrid_state/main)
+
+Outside of this there are many more details to discover that will be skipped over for now:
+- Private and public contexts
+- Encryption keys and events
+- Oracle's role in using notes
+- Value Serialization/Deserialization
+
+This explainer will go through some examples that touch on what has been covered [here](../../../../learn/concepts/storage/notes).
+
+#### Some code context
+The way Aztec (`aztec-packages` monorepo) benefits from the Noir language is via three important components:
+- `Aztec-nr` - a Noir framework enabling contracts on Aztec, written in Noir. Includes useful Note implementations.
+- `noir contracts` - , example Aztec contracts
```

Comment:
don't really understand why we have to mention the monorepo here

Reply:
Succinctly informing the user of the monorepo and how it contains the listed components.
Also to introduce it prior to linking to files in it.

Reply:
Removed.
From the links that follow, a readers assumption of a mono-repo or not can be confirmed by going to the top of aztec-packages.